debian/changelog

```diff
@@ -1,21 +1,33 @@
-cloud-init (25.1-0ubuntu1~24.04.1) UNRELEASED; urgency=medium
+cloud-init (25.1.1-0ubuntu1~24.04.1) noble; urgency=medium
 
   * Drop cpicks which are now upstream:
     - cpick-d75840be-fix-retry-AWS-hotplug-for-async-IMDS-5995
     - cpick-84806336-chore-Add-feature-flag-for-manual-network-waiting
-  * refresh patches:
+    - d/p/cpick-c60771d8-test-pytestify-test_url_helper.py
+    - d/p/cpick-8810a2dc-test-Remove-CiTestCase-from-test_url_helper.py
+    - d/p/cpick-582f16c1-test-add-OauthUrlHelper-tests
+    - d/p/cpick-9311e066-fix-Update-OauthUrlHelper-to-use-readurl-exception_cb
+  * refresh patches
```

Comment:
Did we intentionally remove the extra refresh section rather than combining it? Was there something wrong with those?